includes/class-sensei-admin.php

```diff
@@ -130,6 +131,22 @@ public function add_course_order() {
 		);
 	}
 
+	/**
+	 * Add Lesson order page to admin panel.
+	 *
+	 * @since  4.0.0
+	 */
+	public function add_lesson_order() {
+		add_submenu_page(
+			null,
+			__( 'Order Lessons', 'sensei-lms' ),
+			__( 'Order Lessons', 'sensei-lms' ),
+			'manage_sensei',
```

Comment:
This should probably be `edit_published_lessons` [as it is currently](https://github.com/Automattic/sensei/blob/4aa6c35cce896ebb38f2231a3bbfc2ed40857a80/includes/class-sensei-admin.php#L112).

Reply:
Fix: https://github.com/Automattic/sensei/pull/4687/commits/b6e5dcbc1a59a9b20f955b6c360d0ec968e56377#diff-a700db88caca1edaa7ce24fc214d9344458fb9a1a636e4f8fa6bfb9eaa9a93dfR146